backend/src/api/auth.ts

```diff
@@ -153,6 +155,9 @@ export const callback = async (event, context) => {
     await user.save();
   }
 
+  user.lastLoggedIn = new Date(new Date().getTime());
```

Comment:
Let's do:

```suggestion
  user.lastLoggedIn = new Date(Date.now());
```

for consistency with elsewhere in the codebase for getting the current time

Reply:
sounds good, i'll make those changes